server.go

```diff
@@ -507,9 +514,66 @@ func (s *Server) checkMaxSlices(scheme string, hostPort string) (map[string]uint
 	return pb.MaxSlices, nil
 }
 
+// monitorDiagnostics periodically polls the the Pilosa Indexes for cluster info.
+func (s *Server) monitorDiagnostics() {
+	if s.DiagnosticInterval <= 0 {
+		s.Logger().Printf("diagnostics disabled")
```

Comment:
Might be a good idea to write out a message here if diagnostics is enabled, so it's transparent to the user. Maybe a little blurb with a link to the documentation would be nice, ie. "Pilosa is currently configured to send small diagnostics reports every hour to our team. More information here: http...."

Reply:
done

Reply:
"the the"